### DISCUSSION TITLE
Can you make a version without the commands already present in classic minecraft

### DISCUSSION THREAD

**Discussion by freerig** [EssentialsX/Essentials#4898] [Ideas and Feature Suggestions]
Hello,
I think the title is quite explicit,
can you make a version for 1.16.5 without the commands already present in the classic minecraft, because I would like to use a good autocompletion of the commands like /kill, /kick, etc.
Thank you in advance

**Comment by osfanbuff63** [comment]
This can be done using the `commands.yml` file on your server (which I have to assume is Bukkit-based since, well, this is EssX :P). Paste this in:

```yml
aliases:  
  kick:
  - minecraft:kick $1-
```

You can replace `kick` with any other command to override, however, make sure the `minecraft:` stays. `aliases` is only needed once.

**Reply by freerig** [reply]
hank you very much, it really helps me, but I won't to know if it's possible to have the auto-completion with this method

**Reply by kennyrkun** [reply]
I do believe that this method will allow using tab-completers, but if it does not, another option is to add the commands you don't want to use Essentials for to the `disabled-commands` section of the Essentials `config.yml`.
